CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 
 ## Recent Changes
 
+<<<<<<< HEAD
+- Bug: Default encoding for I/O operations should be UTF-8 
+- Feature: Added method to load profile properties from environment variables
+=======
```

Comment:
there is still a merge conflict here 😋

Reply:
Yes my bad, I forgot to resolve them :yum: